config.json

```diff
@@ -458,14 +463,24 @@
           }
         },
         "t": {
-          "description": "Generates project.jsons for test projects, restores packages, builds product and then builds tests against the generated project.json files.",
+          "description": "Generates project.jsons for test projects, restores packages.",
           "settings": {
             "RestoreDuringBuild":  true,
             "BuildTestsAgainstPackages": true,
             "BatchGenerateTestProjectJsons":  "default",
             "BatchRestorePackages": "default"
           }
         },
+        "tl": {
```

Comment:
Why?  Aligning build pipeline with the same process devs use where possible is a definite goal.